src/Decoder.c

```diff
@@ -2313,13 +2313,24 @@ static void ZydisSetAttributes(ZydisDecoderContext* context, ZydisDecodedInstruc
             default:
                 break;
             }
-        } else
+        }
+
+        if (def->accepts_NOTRACK)
         {
-            if (def->accepts_segment)
+            instruction->attributes |= ZYDIS_ATTRIB_ACCEPTS_NOTRACK;
+            if (context->decoder->decoder_mode[ZYDIS_DECODER_MODE_CET] &&
+                (context->prefixes.effective_segment == 0x3E))
```

Comment:
Note: `effecctive_segment` always prioritizes `0x64`/`0x65` in 64-bit mode which is perfect for our case regarding Intel SDM:

> In 64-bit mode, the 3EH prefix on an indirect call or jmp is recognized as a no-track prefix when the following 
conditions are satisfied. 
> -  3EH must be the last legacy prefix of any group (except any REX). 
> -  There must not be a 64H/65H prefix on the instruction. 
>
> In legacy/compatibility mode, the 3EH prefix on an indirect call or jmp is recognized as a no-track prefix 
when it is the last group 2 prefix on the instruction.